src/convert/transform-react-node.ts

```diff
@@ -0,0 +1,36 @@
+import traverse from "@babel/traverse";
+import { TransformerInput } from "./transformer";
+import * as t from "@babel/types";
+import { replaceWith } from "./utils/common";
+
+/**
+ * The codemod properly transforms the flow type of React.Node to typescript type React.ReactNode.
```

Comment:
This makes since if you realize Greenhouse devs added this transform, maybe update the language so that’s not necessary? “So far we have transformed [….] Now we transform direct imports”